src/Workspaces/Core/Portable/FindSymbols/Shared/AbstractSyntaxIndex_Persistence.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Microsoft.CodeAnalysis.FindSymbols;
 
-internal partial class AbstractSyntaxIndex<TIndex>
+internal abstract partial class AbstractSyntaxIndex<TIndex>
```

Comment:
partial abstract types had all their parts updated to be 'abstract' to make it easy to see thast this is abstract (and not accidentally seal it).  Same with partial statics.